src/Servers/HttpSys/src/NativeInterop/ComNetOS.cs

```diff
@@ -12,11 +11,18 @@ internal static class ComNetOS
         // Minimum support for Windows 7 is assumed.
         internal static readonly bool IsWin8orLater;
 
+        internal static bool SupportsGoAway { get; }
+
         static ComNetOS()
         {
             var win8Version = new Version(6, 2);
 
             IsWin8orLater = (Environment.OSVersion.Version >= win8Version);
+
+            var win1019H2Version = new Version(10, 0, 18363);
```

Comment:
Shouldn't the check be against `new Version(6, 3, 18363)`?

Reply:
No, this is the real version number and it does work if you have an app.manifest. Was just talking to glenn about doing a more direct version check (e.g. the registry).

Reply:
No, this is the real version number and it does work if you have an app.manifest. Was just talking to glenn about doing a more direct version check (e.g. the registry).

Reply:
reallllllyyy... I didn't know that.

Reply:
This may be good enough for preview1.

Next step: I'm going to create a new shared source library in extensions that we can use to do these OS version checks. We'll need it for the test skip conditions and here.